app/assets/stylesheets/pages/involvement.scss

```diff
@@ -0,0 +1,89 @@
+/* Involvement Styles */
+
+.involvement_page {
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    padding-left: 5cm;
+    padding-right: 5cm;
+    margin-top: 2cm;
+  }
+
+  .title {
+    color: $orange;
+    font-size: $font-size-l;
+  }
+
+  .optionsContainer {
+    display: flex;
+    flex-direction: row;
+    margin-top: 50px;
+  }
+
+  .option {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    height: 300px;
+  }
+
+  .middle {
+    margin-left: 15px;
+    margin-right: 35px;
+  }
+
+  .optionIcon {
+    display: flex;
+    align-self: center;
+    justify-content: center;
+    height: 75px;
+    position: relative;
+    img {
+      height: 100%;
+    }
+  }
+
+  .optionTitle {
+    color: $orange;
+    font-size: $font-size-s;
+  }
+
+  .separator {
+    border-bottom: 1px $orange solid;
+  }
+
+  .optionDescription {
+    margin-top: 15px;
+  }
+
+  .optionButton {
+    align-self: center;
+    margin-top: auto;
+  }
+}
+
+  /* Footer */
+
+.involvement_footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  width: 100%;
+  background-color: $orange;
+  color: $white;
+
+  .footer__links-container {
```

Comment:
usually when you use the __  in a css selector, you usually have an element named with whatever is before the __. 

So change .footer__links-container to .involvement-footer__links-container or simplify involvement-footer to something else